browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html

```diff
@@ -28,6 +48,58 @@
         </div>
       </div>
     </div>
+    <div class="settings-box">
+      <div class="flex">
+        <div class="label shields-primary-title">$i18n{defaultView}</div>
+        <settings-radio-group
+          pref="{{prefs.brave.advanced_view_default}}">
+          <controlled-radio-button class="shields-view-col" name="false">
+            <div class="flex-center">
+              <svg width="50" height="50" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                <defs>
+                  <path d="M2 0h46a2 2 0 0 1 2 2v46a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2z" id="simpleViewImgPath"/>
+                </defs>
+                <g fill="none" fill-rule="evenodd">
+                  <mask id="b" fill="#fff">
```

Comment:
nit: svg mask with ID being used, but perhaps not necessary and could cause issues

Reply:
updated